### PR TITLE
Issue #134: Support non-Backdrop projects.

### PR DESCRIPTION
**Pull request by ghost**
Addresses https://github.com/backdrop-contrib/drush/issues/134

### CODE REVIEW

File: commands/make/bmake.drush.inc [hunk 1]
```diff
@@ -62,37 +62,48 @@ function bmake() {
...
+    else {
+
+      // $details is an array.
+      if (!empty($details['git'])) {
+        // Clone a git repository.
+        $tags = (object) array(
+          'type' => 'project_' . rtrim($section, 's'),
+        );
+        $path = backdrop_pm_get_path($tags);
+        passthru("git clone $details[git] $path/$project");
```
Comment (serundeputy) [inline commands/make/bmake.drush.inc:109]: i think we'll need a `bmake_validate()` function here to check if the server has git installed. something like https://github.com/backdrop-contrib/drush/blob/1.x-1.x/commands/pm/backdrop_pm.drush.inc#L257

File: commands/make/bmake.drush.inc [hunk 2]
```diff
@@ -62,37 +62,48 @@ function bmake() {
...
+    }
+    else {
+
+      // $details is an array.
+      if (!empty($details['git'])) {
+        // Clone a git repository.
+        $tags = (object) array(
+          'type' => 'project_' . rtrim($section, 's'),
+        );
+        $path = backdrop_pm_get_path($tags);
```
Comment (serundeputy) [inline commands/make/bmake.drush.inc:108]: do we have access to the `backdrop_pm_get_path` function here or do we have to include the pm file?

File: commands/make/bmake.drush.inc (hunk 1 above)
Comment (ghost) [reply commands/make/bmake.drush.inc:109]: Good idea! See this new commit: https://github.com/backdrop-contrib/drush/pull/187/commits/a5498289a0639b719efbc82e3b5d5acdc446efec

File: commands/make/bmake.drush.inc (hunk 2 above)
Comment (ghost) [reply commands/make/bmake.drush.inc:108]: I think we have access. I mean, it works, insomuch that it returns a path... No errors or anything.

**Review by serundeputy — APPROVED** [review]

**Pull request merged**